spezi-module/onboarding/src/main/java/edu/stanford/spezikt/spezi_module/onboarding/onboarding/Area.kt

```diff
@@ -0,0 +1,13 @@
+package edu.stanford.spezikt.spezi_module.onboarding.onboarding
+
+/**
+ * Represents an area of the onboarding screen.
+ * @property title The title of the area.
+ * @property iconId The resource ID for the icon.
+ * @property description The description of the area.
+ */
+data class Area(
+    val title: String,
+    val iconId: Int, // Resource ID for the icon
```

Comment:
Instead of allowing for an arbitrary resource icon Id, I would introduce an enum for our supported types, and move the drawables from design module to onboarding

Reply:
I think that we do provide a few icons through the design module, but basically I think that users of the framework want to be able to use their own icons and not be limited to a predefined selection, right?